packages/core/src/service/services/customer.service.ts

```diff
@@ -668,15 +673,15 @@ export class CustomerService {
             type: HistoryEntryType.CUSTOMER_ADDRESS_CREATED,
             data: { address: addressToLine(createdAddress) },
         });
-        this.eventBus.publish(new CustomerAddressEvent(ctx, createdAddress, 'created'));
+        this.eventBus.publish(new CustomerAddressEvent(ctx, createdAddress, 'created', input));
         return createdAddress;
     }
 
     async updateAddress(ctx: RequestContext, input: UpdateAddressInput): Promise<Address> {
-        const address = await this.connection.getEntityOrThrow(ctx, Address, input.id, {
+        const address = await this.connection.getEntityOrThrow<Address>(ctx, Address, input.id, {
```

Comment:
The generic argument here should not be needed. Ditto on line 684, 717, 720.

Reply:
That's true. Maybe my IDE was just wrong ^^